android/.gitignore

```diff
@@ -1,9 +1,13 @@
 *.iml
-.gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
+/.idea/*
+gradle-wrapper.jar
+/.gradle
+/captures/
+/gradlew
+/gradlew.bat
```

Comment:
gradle wrapper needs not to be ignored. Look for standard java gitignore and take it from there